com.github.k4zmu2a.spacecadetpinball.metainfo.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+    <id>com.github.k4zmu2a.spacecadetpinball</id>
+    <name>3D Pinball for Windows – Space Cadet</name>
+    <display_name>Pinball</display_name>
+    <summary>Begins a game of 3-D Pinball.</summary>
+    <url type="homepage">https://github.com/k4zmu2a/SpaceCadetPinball</url>
+    <url type="bugtracker">https://github.com/k4zmu2a/SpaceCadetPinball/issues</url>
+    <description>
+        <p>
+            3D Pinball for Windows – Space Cadet is a pinball video game developed by Cinematronics and published by Maxis in 1995. It features pre-rendered 3D graphics and three tables—Space Cadet, Skullduggery, and Dragon's Keep. On each table, there are displays on the side that show the players' score, ball number, player number, a display for various information and a table-specific image.
```

Comment:
this description feel inacurate. Isn't that a reimplementation of an older pinball game for windows, that also reuse the original assets?
The description read like it's a Windows game.

Reply:
Yeah, but I wanted to try to keep the "original" feel by using the same description. But there was no original description for the installer